### PR TITLE
Added Manifest file format

### PR DESCRIPTION
**Pull request by gebansal**

### CODE REVIEW

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  "description": "A plain-text description",
+  "icons": {...},
+
+  // Optional
+  "action": ...,
+  "author": ...,
+  "automation": ...,
+  "background": {
+    // Required
+    "service_worker": ...
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:46]: added ... like v2

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    "matches": ["*://*.example.com/*"]
+  },
+  "file_browser_handlers": [...],
+  "file_system_provider_capabilities": {
+    "configurable": true,
+    "multiple_mounts": true,
+    "source": "network"
+  },
+  "homepage_url": "http://path/to/homepage",
+  "host_permissions": [...],
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:70]: Noting a diff: the v2 listing lacks [ ] square brackets

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+  "requirements": {...},
+  "sandbox": [...],
+  "short_name": "Short Name",
+  "storage": {
+    "managed_schema": "schema.json"
+  },
+  "system_indicator": ...,
+  "tts_engine": {...},
+  "update_url": "http://path/to/updateInfo.xml",
+  "version_name": "aString",
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:101]: Noting a diff: v2 listing has:
"version_name": ...,

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+*  The permissions that are needed for the extension to run.
+
+The format for `manifest.json` for extensions is moving from Manifest V2 to Manifest V3.  Both formats are shown here.  To migrate a Manifest V2 extension to Manifest V3, navigate to [Prepare to update your extensions from Manifest v2 to v3][MigrateToMV3].
+
+
+## Format of manifest\.json for extensions using Manifest V3
+
+The following pseudocode shows the fields that are supported in `manifest.json` for extensions, for a Manifest V3 package.
+
+```json
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:27]: You can see my minor tweaks to the JSON listings at https://github.com/MicrosoftDocs/edge-developer/pull/1267/commits/0587d65abc99089bb8de8e252392a29a782de51b

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  "default_locale": "en",
+  "description": "A plain-text description",
+  "icons": {...},
+
+  // Optional
+  "action": ...,
+  "author": ...,
+  "automation": ...,
+  "background": {
+    // Required
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:47]:
```suggestion
    // If `background` is included, `service_ worker` is required
```

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+title: Manifest file format
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 06/07/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, web development, html, css, javascript, developer, extensions, mv2, mv3, manifest
+---
+
+# Manifest file format
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:12]:
```suggestion
# Manifest file format for extensions
```

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+  "manifest_version": 2,
+  "name": "My V2 Extension",
+  "version": "versionString",
+
+  // Recommended
+  "default_locale": "en",
+  "description": "A plain-text description",
+  "icons": {...},
+
+  // Pick one (or none)
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:126]:
```suggestion
  // Pick one or none
```

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+  "browser_action": {...},
+  "page_action": {...},
+
+  // Optional
+  "action": ...,
+  "author": ...,
+  "automation": ...,
+  "background": {
+    // Recommended
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:135]:
```suggestion
    // If `background` is included, the following is recommended
```

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+    "persistent": false,
+    // Optional
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:137]:
```suggestion
    // If `background` is included, the following is optional
```

File: microsoft-edge/extensions-chromium/getting-started/manifest-format.md [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+  "content_scripts": [{...}],
+  "content_security_policy": "policyString",
+  "converted_from_user_script": ...,
+  "current_locale": ...,
+  "declarative_net_request": ...,
+  "devtools_page": "devtools.html",
+  "differential_fingerprint": ...,
+  "event_rules": [{...}],
+  "externally_connectable": {
+    "matches": ["*://*.example.com/*"]
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/getting-started/manifest-format.md:153]:
```suggestion
    "matches": ["*://*.contoso.com/*"]
```

**Review by Reezaali — CHANGES_REQUESTED** [review]
Please review my comments below. Thank you.

**Review by Reezaali — APPROVED** [review]

**Pull request merged**